server/src/lib/dokku/apps/delete.ts

```diff
@@ -1,13 +1,13 @@
 import NodeSsh from 'node-ssh';
 
-// named deleteApp instead of delete because of the "strict" warning
-// apparently you can't use word "delete" as var name
-export const deleteApp = async (ssh: NodeSsh, appName: string) => {
-  const resultAppsDelete = await ssh.execCommand(
+export const destroy = async (ssh: NodeSsh, appName: string) => {
```

Comment:
should the name of the file change to destroy too? (not a blocker for merging can be done later :) )